### ISSUE TITLE
User list clean on old site - what to wipe

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#478]
Just having a look on the existing old site at the user list:

![image](https://cloud.githubusercontent.com/assets/6060985/5978811/88fdb554-a89a-11e4-950a-ec5f0e469aca.png)

I need to know exactly what each column means. If the user is "active"but login states "never"  that means, I assume that their email authentication went through OK but they never bothered to actually login in?

If login states "never" can we safely assume that they are not interested as such? Will their email details have been sent to Mailchimp and therefore they are on the mailing list? 

At what point does Drupal send mailchimp any message to delete a user from mailchimp i.e. the user has deleted their account on Drupal?

At the moment Drupal uploads the user info to mailchimp to a list called "Herne Hill Forum list from new web site". We then manually combine this list with the master list. I think the new site can upload direct into the master list, we trust it now. 

Is there a way I can manually trigger the upload to mailchimp routine so I can make some tests? I think it happens every hour or so but testing what happens and where would be rather slow waiting for each upload...

**Comment by pmackay** [comment]
Do you mean "last access" when you say "login"?

Can users currently create accounts without email approval?

I dont know about Mailchimp, can you cross-check?

I would expect if you delete a user it would also delete from Mailchimp. Might need to run cron first. 

If its triggered by cron then you can run that by going to the Home menu in top left corner and selecting Run cron.

**Comment by djwesto** [comment]
This is why I was a little iffy the other day about you just going in and deleting user accounts.

People might register on your site for all sorts of reasons. How confident are you that all those users who've never logged in might not want to at some point the future (i.e. when we launch their snazzy new site) and then they'll discover their user account has been deleted.

**Comment by GilesGibson** [comment]
True, I don't want to delete any potential user if I can. However, apart from the obvious spammers who just get deleted (and content) there may be hundreds who signed up, forgot their password and then created another login later, or who just are not interested any more. 

Not sure how best to do user maintenance on the site, need to do something otherwise over time the user list gets more and more stuffed with "dead" users. 

@pmackay Re: last access - yes, the site details when the user last logged in. 

Re: Cron - is there any issue with running CRON? last time it removed the maintenance menu from my screen which meant I couldn't easily get to the user list.

**Comment by djwesto** [comment]
Perhaps could you email all the users who've never logged in and ask them to log in and update their details if they want to keep the account (therwise, you'll delete them in 30 days time) or something like that?

**Comment by GilesGibson** [comment]
be a good idea, no idea if it is possible to select that group of users for an mass email though.

**Comment by pmackay** [comment]
@GilesGibson I think "never" means they've created accounts but then never logged in again (https://www.drupal.org/node/70346). Up to you if you want to clean them out.

Suggest we review mailchimp after migration and ensure it syncs with user list, but you could check now about when users are deleted. BUT if you have problems running cron, maybe best to leave til after migration?

**Comment by GilesGibson** [comment]
OK, lets bring them across and then do a cull. We have better ways of emailing select groups on the new site I think/hope.

**Comment by GilesGibson** [comment]
Need to know about user admin. I can update admin manual if you let us know where to look within the site.

**Comment by pmackay** [comment]
@GilesGibson what do you want to do with this one now?

**Comment by GilesGibson** [comment]
I would be inclined to clear off all users that are not active. Not sure why we have about 100 or more like that but maybe they never bothered to update their account. I suspect that they have unsubscribed from mailchimp anyway.
